src/os/command/feature/featurefillcolorcmd.js

```diff
@@ -0,0 +1,119 @@
+goog.provide('os.command.FeatureFillColor');
+
+goog.require('os.command.AbstractFeatureStyle');
+goog.require('os.events.PropertyChangeEvent');
+goog.require('os.metrics');
+
+
+
+/**
+ * Changes the fill color of a feature
+ * @extends {os.command.AbstractFeatureStyle}
+ * @param {string} layerId
+ * @param {string} featureId
+ * @param {Array<number>|string} color
+ * @param {(Array<number>|string)=} opt_oldColor
+ * @constructor
+ */
+os.command.FeatureFillColor = function(layerId, featureId, color, opt_oldColor) {
```

Comment:
It seems there is a lot of duplicate code between this and `os.command.FeatureColor`. Same goes for the opacity commands and the stroke color/opacity. We should either abstract out the common logic, or the fill/stroke variants should extend the originals and replace only what they need. You're welcome to refactor the originals to make this easier/cleaner. It seems with a few small changes we could significantly reduce the code duplication in this PR.

Reply:
On second thought, it seems entirely possible to add an option to the constructor params (`opt_isFill`) that changes the operation of the command to change the fill color/opacity.

Reply:
I have consolidated the commands and removed the separate Stroke and Fill command files. These are still separate commands and still show up in action history as separate entries, but should only affect the fill, stroke, or both, as dictated by the user's action.